virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -194,73 +201,48 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
-
-//BZ2216038 from Ying
-* If you simultaneously clone more than 1000 VMs using the provided DataSources in the `openshift-virtualization-os-images` namespace, it is possible that not all of the VMs will move to a running state. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2216038[*BZ#2216038*])
-** As a workaround, deploy VMs in smaller batches.
 
-//BZ2247593 from Ying
-* Live migration cannot be enabled for a virtual machine instance (VMI) after a hotplug volume has been added and removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055595[*BZ#2247593*])
 
 [discrete]
 [id="virt-4-15-ki-virtualization"]
 ==== Virtualization
-//4.14 Leave in per Stu
-* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037611[*BZ#2037611*])
+//4.15 still unresolved
+* When adding a virtual Trusted Platform Module (vTPM) device to a Windows VM, the BitLocker Drive Encryption system check passes even if the vTPM device is not persistent. This is because a vTPM device that is not persistent stores and recovers encryption keys using ephemeral storage for the lifetime of the `virt-launcher` pod. When the VM migrates or is shut down and restarts, the vTPM data is lost. (link:https://issues.redhat.com/browse/CNV-36448[*CNV-36448*])
+
+//4.15 still unresolved 
+* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://issues.redhat.com/browse/CNV-33835[*CNV-33835*])
 ** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
 
-// Leave in for 4.14 per Stu; remove from 4.15 RNs and move content to CNV docs, per Avital
-* With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS `Extended Master Secret` (EMS) extension (link:https://datatracker.ietf.org/doc/html/rfc7627[RFC 7627]) is mandatory for TLS 1.2 connections on FIPS-enabled RHEL 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2157951[*BZ#2157951*])
+//4.15 Keep this per Stu, ETA for fixing this issue is 4.18. 
+* With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS `Extended Master Secret` (EMS) extension (link:https://datatracker.ietf.org/doc/html/rfc7627[RFC 7627]) is mandatory for TLS 1.2 connections on FIPS-enabled RHEL 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected. 
```

Comment:
Just curious, is there a reason that EMS is not in literals when the written out term is?
```suggestion
* With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS `Extended Master Secret` (`EMS`) extension (link:https://datatracker.ietf.org/doc/html/rfc7627[RFC 7627]) is mandatory for TLS 1.2 connections on FIPS-enabled RHEL 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected. 
```

Reply:
This is pre-existing content from 4.14.